internal/operators/ocs/validations.go

```diff
@@ -193,42 +202,45 @@ func (o *operator) canOCSBeDeployed(hosts []*models.Host, nodeInfo *resourceInfo
 	return true, status
 }
 
-func validateRequirementsForMode(o *operator, nodeInfo *resourceInfo, mode ocsDeploymentMode) bool {
+func validateRequirementsForMode(o *operator, ocsClusterResources *ocsClusterResourcesInfo, mode ocsDeploymentMode) bool {
 	var totalCPUs int64
 	var totalRAM int64
 
-	if mode == compactMode || mode == standardMode {
-		totalCPUs = o.config.OCSStandardAndCompactModeCPUCount
-		totalRAM = o.config.OCSStandardAndCompactModeRAMGiB
+	if mode == compactMode {
+		totalCPUs = o.config.OCSCompactModeClusterCPUCount + o.config.OCPMinCPUCoresMaster*ocsClusterResources.numberOfHosts
+		totalRAM = o.config.OCSCompactModeClusterRAMGiB + o.config.OCPMinRamGibMaster*ocsClusterResources.numberOfHosts
+	} else if mode == minimalMode {
+		totalCPUs = o.config.OCSMinimalModeClusterCPUCount + o.config.OCPMinCPUCoresWorker*ocsClusterResources.numberOfHosts
+		totalRAM = o.config.OCSMinimalModeClusterRAMGiB + o.config.OCPMinRamGibWorker*ocsClusterResources.numberOfHosts
 	} else {
-		totalCPUs = o.config.OCSMinimumCPUCount
-		totalRAM = o.config.OCSMinimumRAMGiB
+		totalCPUs = o.config.OCSStandardModeClusterCPUCount + o.config.OCPMinCPUCoresWorker*ocsClusterResources.numberOfHosts
```

Comment:
While this will evaluate the expression the intended way, to make it easier to read please use () for all the expressions:
Eg: 

totalCPUs = o.config.OCSMinimalModeClusterCPUCount + (o.config.OCPMinCPUCoresWorker*ocsClusterResources.numberOfHosts)

Reply:
Done